2.7/Dockerfile

```diff
@@ -7,7 +7,7 @@ MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
 EXPOSE 8080
 
 ENV PYTHON_VERSION=2.7 \
-    PATH=$HOME/.local/bin/:$PATH \
+    PATH=$HOME/.local/bin/:/opt/app-root/node_modules/.bin:$PATH \
```

Comment:
Is there a reason to use `/opt/app-root/node_modules/.bin` instead of `$HOME/node_modules/.bin/:$HOME/.npm-global/bin/` as it is used in [s2i-nodejs-container](https://github.com/sclorg/s2i-nodejs-container/blob/master/4/Dockerfile#L18)?